### DISCUSSION TITLE
Congratulations to Adil

### DISCUSSION THREAD

**Discussion by MaxHalford** [online-ml/river#1033] [Show and tell]
I want to congratulate @AdilZouitine for his effort implementing statistics in Rust. His work has just been shipped in version 0.13. Now users will benefit from super fast implementations for some statistics.

This is really cool! Adil did a lot of work, and has also laid down the foundation for implementing more stuff in Rust if necessary. Thanks Adil! @online-ml/devs we can give him a round of applause for this grueling work :)

**Comment by smastelini** [comment]
👏  and congrats, too, on the Neurips paper!

**Reply by smastelini** [reply]
Now that the heavy lifting has been done, I may try to learn a thing or two in Rust. The speed-ups are crazy!! Congrats @AdilZouitine!

**Reply by AdilZouitine** [reply]
Thank you !! 😄  Coding in rust is a great experience, the ecosystem is top-notch!

**Comment by AdilZouitine** [comment]
Thanks team ❤️
